_data/events_gdcr2018/Zaragoza.json

```diff
@@ -0,0 +1,15 @@
+{
+    "title": "GDCR18 en Zaragoza",
+    "url": "https://www.meetup.com/es-ES/agilearagon/?_cookie-check=dq2JuzuNxh_DBw19",
+    "moderators": [],
```

Comment:
Isn't there already someone who will host it?

Reply:
Not in this moment. But maybe best option will be myself. I will add .Thanks